src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/Directory.Build.targets

```diff
@@ -1,32 +1,42 @@
 <Project>
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)..\'))" />
 
   <PropertyGroup>
-    <_TypescriptRootPath>$(MSBuildThisFileDirectory)/TypeScript</_TypescriptRootPath>
-    <_NpmInstallStampFile>$(_TypescriptRootPath)/node_modules/.installed</_NpmInstallStampFile>
+    <_TypescriptRootPath>$(MSBuildThisFileDirectory)TypeScript</_TypescriptRootPath>
+    <_NpmInstallStampFile>$(_TypescriptRootPath)\node_modules\.installed</_NpmInstallStampFile>
   </PropertyGroup>
 
   <Target Name="NpmInstall"
-          Condition="'$(EnableEvaluationReportBuild)'=='true'"
```

Comment:
Was removing the condition intentional? Won't this fail on machines where node is missing?

Reply:
Ah looks like `BuildTypeScriptBundle` is still conditioned and looks like this target can only be called via that one... So this is probably fine.

Reply:
Yes it was intentional since it was not necessary.